test/libyul/KnowledgeBaseTest.cpp

```diff
@@ -99,30 +99,24 @@ BOOST_AUTO_TEST_CASE(difference)
 		let e := sub(c, 12)
 	})");
 
-	BOOST_CHECK(
-		kb.differenceIfKnownConstant("c"_yulstring, "b"_yulstring) ==
+	BOOST_CHECK(kb.differenceIfKnownConstant("c"_yulname, "b"_yulname) ==
 		u256(20)
 	);
-	BOOST_CHECK(
-		kb.differenceIfKnownConstant("b"_yulstring, "c"_yulstring) ==
+	BOOST_CHECK(kb.differenceIfKnownConstant("b"_yulname, "c"_yulname) ==
 		u256(-20)
 	);
-	BOOST_CHECK(!kb.knownToBeDifferentByAtLeast32("b"_yulstring, "c"_yulstring));
-	BOOST_CHECK(kb.knownToBeDifferentByAtLeast32("b"_yulstring, "d"_yulstring));
-	BOOST_CHECK(kb.knownToBeDifferentByAtLeast32("a"_yulstring, "b"_yulstring));
-	BOOST_CHECK(kb.knownToBeDifferentByAtLeast32("b"_yulstring, "a"_yulstring));
+	BOOST_CHECK(!kb.knownToBeDifferentByAtLeast32("b"_yulname, "c"_yulname));
+	BOOST_CHECK(kb.knownToBeDifferentByAtLeast32("b"_yulname, "d"_yulname));
+	BOOST_CHECK(kb.knownToBeDifferentByAtLeast32("a"_yulname, "b"_yulname));
+	BOOST_CHECK(kb.knownToBeDifferentByAtLeast32("b"_yulname, "a"_yulname));
 
-	BOOST_CHECK(
-		kb.differenceIfKnownConstant("e"_yulstring, "a"_yulstring) == u256(208)
+	BOOST_CHECK(kb.differenceIfKnownConstant("e"_yulname, "a"_yulname) == u256(208)
 	);
```

Comment:
```suggestion
	BOOST_CHECK(kb.differenceIfKnownConstant("e"_yulname, "a"_yulname) == u256(208));
```

Same for the ones below.